src/main/webapp/resources/js/components/analyses-outputs/SingleSampleAnalysisOutputs.jsx

```diff
@@ -36,9 +37,11 @@ export default function SingleSampleAnalysisOutputs({
   const [selectedRowKeys, setSelectedRowKeys] = React.useState([]);
   const [filteredOutputs, setFilteredOutputs] = React.useState(null);
 
-  // Ant Design Table options
-  const PAGE_SIZE_OPTIONS = [5, 10, 25, 50, 100];
-  const DEFAULT_PAGE_SIZE = 10;
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
+
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(outputs.length);
}, [outputs.length]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b